packages/trip-form/src/DateTimeSelector/index.tsx

```diff
@@ -89,9 +89,17 @@ interface DepartArriveOption {
  * @param {*} type One of the HTML5 input types.
  */
 function isInputTypeSupported(type: string): boolean {
-  const input = document.createElement("input");
-  input.setAttribute("type", type);
-  return input.type === type;
+  let retVal = false;
+  try {
```

Comment:
If `typeof document !== "undefined"` is the differentiator between server-side rendering and browser rendering, then is the try/catch block necessary?

Reply:
You're right Binh .. it was a bit extra, and latest push has removed it.

Reply:
Cool! With the latest changes, you should be able to replace `retVal` with return statements.